include/ringmesh/basic/common.h

```diff
@@ -44,8 +44,18 @@
 #include <ringmesh/ringmesh_config.h>
 
 #if defined( _WIN32 )
-#ifndef WIN32
-#define WIN32
+#define RINGMESH_WINDOWS
+#else
+#undef RINGMESH_WINDOWS
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove this #undef directive. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3APPUndefUsage)